Update 4 (#251)
NilPointer-Software/bdfd-wiki: src/bdscript/authorURL.md

# $authorURL
Adds a hyperlink to the author text.

## Syntax
```
$authorURL[url;(index)]
```
> `$authorURL[]` will not work if there is no text provided in `$author[]`.

### Parameters
- `url`: The link to set as the author hyperlink.

> - `index`: What embed the author url should belong to. (Optional) [(learn more)](../resources/embedIndexes.md)

Reviewer: ```suggestion
- `index`: To which embed the author URL will be added. (Optional) [(learn more)](../resources/embedIndexes.md)
```
